release-notes/ReleaseNoteTemplate.yml

```diff
@@ -0,0 +1,8 @@
+issue_key: <jira issue key> # e.g. NATIVE-587, WEBAPP-42
+show_in_stores: <boolean> # whether this note should be shown in the stores, either true or false
+platforms: # relevant platforms, possible values: android and ios
+  - <platform1>
+  - <platform2>
+en: <your note> # english version of the note
+# optional: the note in other languages:
+de: <your german note> # german version of the note
```

Comment:
This is different to the format we discussed in some grooming: https://issues.integreat-app.de/browse/WEBAPP-536

Is there a reason for this? I think it makes sense to have multiple items per release note.

Reply:
Yep, right. I first wanted to use that format but later figured this just makes more sense. My reasons:
* If we have multiple items, we need to define whether it is a production note or not for each item separately which makes the format more complicated than the one discussed in the grooming.
* Normally there should not be more than one note per issue, but if there are more than one, one can simply add multiple release note files.
* This format is simpler than the one discussed and therefore easier to use for developers.